code/obj/machinery/plantpot.dm

```diff
@@ -84,6 +84,136 @@
 				return ..()
 		..()
 
+/obj/machinery/plantpot/bareplant
+	name = "arable soil"
+	desc = "A small mound of arable soil for planting and plant based activities."
+	anchored = 1
+	mats = 0
+	deconstruct_flags = 0
+	icon_state = null
+	power_usage = 0
+	growth_rate = 1
+	/// plant to grow
+	var/datum/plant/spawn_plant = null
+	/// growth level to spawn with
+	var/spawn_growth = null
+	/// list of commuts to apply to plant
+	var/list/datum/plant_gene_strain/spawn_commuts = list()
+	/// apply seedless plant gene strain
+	var/seedless = null
+	/// apply terminator plant gene strain
+	var/terminator = null
+	/// apply toxic plant gene strain
+	var/toxic = null
```

Comment:
why have the vars for those 3 commuts separate from the commut list?

Reply:
Reduce barrier of entry to using the feature for those with limited hydroponics code knowledge. I have no qualms with it's removal.